api/src/test/java/org/pmiops/workbench/ras/RasLinkServiceTest.java

```diff
@@ -205,15 +210,27 @@ public void setUp() throws Exception {
   }
 
   @Test
-  public void testLinkRasSuccess() throws Exception {
+  public void testLinkRasIdMeSuccess() throws Exception {
+    when(mockOidcClient.codeExchange(AUTH_CODE, REDIRECT_URL, RAS_AUTH_CODE_SCOPES))
```

Comment:
These tests are repetitive and a little hard to read.  I'm not sure this level of detail is required here.  Would you mind making a refactoring attempt?

I'm thinking of something roughly like 
```
mockCodeExchangeResponse(TOKEN_RESPONSE_IAL2);
mockAccessTokenResponse(USERINFO_RESPONSE);
```

Reply:
Good call! That's a great idea!